distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -986,50 +996,90 @@ type Decimal
              Decimal.parse "123.456.789,87654" locale=Locale.italy
              # => 123456789.87654
     parse : Text -> Locale | Nothing -> Decimal ! Number_Parse_Error
-    parse text locale:(Locale | Nothing)=Nothing = case locale of
-        Nothing -> Decimal.from_string text
+    parse text locale:(Locale | Nothing)=Nothing -> Decimal ! Number_Parse_Error = case locale of
+        Nothing -> Decimal.from_text text
         Locale.Value java_locale -> Panic.catch ParseException ((NumberFormat.getInstance java_locale).parse text) _->
             Error.throw (Number_Parse_Error.Error text)
 
     ## PRIVATE
     precision : Integer
-    precision self = self.big_decimal.precision
+    precision self -> Integer = self.big_decimal.precision
 
     ## PRIVATE
     scale : Integer
-    scale self = self.big_decimal.scale
+    scale self -> Integer = self.big_decimal.scale
 
     ## PRIVATE
     with_scale : Integer -> Decimal
-    private with_scale self new_scale:Integer =
+    private with_scale self new_scale:Integer -> Decimal =
         if self.scale == new_scale then self else
             Decimal.Value (self.big_decimal.setScale new_scale)
 
     ## PRIVATE
     unscaled_value : Integer
-    unscaled_value self = self.big_decimal.unscaledValue
+    unscaled_value self -> Integer = self.big_decimal.unscaledValue
 
     ## PRIVATE
     internal_representation : [Integer]
-    internal_representation self = [self.unscaled_value, self.precision, self.scale]
+    internal_representation self -> [Integer] = [self.unscaled_value, self.precision, self.scale]
```

Comment:
I don't think this is still valid. Probably accepted by the compiler, but I have no idea what it will do.

I think we should use
```suggestion
    internal_representation self -> Vector Integer = [self.unscaled_value, self.precision, self.scale]
```
instead, as that's what is used most of the time.

Reply:
This is what happens when you write in other languages on the weekend.